R/cov_struct.R

```diff
@@ -133,6 +133,9 @@ COV_TYPES <- local({  # nolint
 #' from SAS `PROC MIXED` refers to heterogeneous ante-dependence and the
 #' homogeneous version is not available in SAS.
 #'
+#' Note that for all non-spatial covariance structures, the time variable must
```

Comment:
maybe add this and above note under `@note` section

Reply:
Good point, done.